### PR TITLE
Uwa 18

### PR DESCRIPTION
**Pull request by rgangineni02**
added the location filter for both moving and landscaping services .check it once for any suggestion or give an approval to merge.

### CODE REVIEW

**Comment by satyanagkumar** [comment]
Is it filtering the services based on location?

**Comment by rgangineni02** [comment]
yes ,added only two locations right now

File: views/landscaping.ejs
Comment (satyanagkumar) [inline views/landscaping.ejs:1]: Yes, Now the code is filtering the services based on location seleced by the user

**Review by satyanagkumar — APPROVED** [review]
Good, this code now filter the moving and landscape services based on location

Thank you.

**Pull request merged**